x/btcstaking/keeper/hooks.go

```diff
@@ -22,18 +21,5 @@ func (k Keeper) Hooks() Hooks {
 
 // AfterSluggishFinalityProviderDetected updates the status of the given finality provider to `sluggish`
 func (h Hooks) AfterSluggishFinalityProviderDetected(ctx context.Context, fpPk *bbntypes.BIP340PubKey) error {
```

Comment:
Do we want to change another word other than sluggish?

Reply:
I'm not sure. The logic here is that `sluggish detected` -> `jail fp` -> `slash rewards`. Do you have something in mind?